pkg/liqo-controller-manager/shadowpod-controller/shadowpod_controller.go

```diff
@@ -113,6 +118,13 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 		return ctrl.Result{}, err
 	}
 
+	// Update ShadowPod status same as Pod status
+	shadowPod.Status.Phase = vkv1alpha1.ShadowPodUnknown
+	if newErr := r.Client.Status().Update(ctx, &shadowPod); newErr != nil {
+		klog.Error(newErr)
+		return ctrl.Result{}, newErr
+	}
```

Comment:
Is this update required? I mean, tecnically this Reconcile should be triggered very soon by pod creation/updates and we will set the value when this happens

Reply:
Yes technically it's not required to update the status, just in case anything happens so updating. Will remove it. Thanks